android/app/src/main/java/com/reactnativenavigation/activities/BottomTabActivity.java

```diff
@@ -64,10 +64,10 @@ private void setupPages(ArrayList<Screen> screens) {
 
     private void setupToolbar(ArrayList<Screen> screens) {
         Screen initialScreen = screens.get(0);
-        setNavigationStyle(initialScreen);
         mToolbar.setScreens(screens);
         mToolbar.setTitle(initialScreen.title);
         setSupportActionBar(mToolbar);
+        setNavigationStyle(initialScreen);
     }
```

Comment:
This had to be at the bottom to ensure the action bar was set before applying styles (else you can't show/hide).